plugins/faustwp/includes/replacement/callbacks.php

```diff
@@ -70,7 +88,6 @@ function image_source_replacement( $content ) {
 	return preg_replace( $patterns, "src=\"{$site_url}/", $content );
 }
 
-add_filter( 'wp_calculate_image_srcset', __NAMESPACE__ . '\\image_source_srcset_replacement' );
```

Comment:
Still testing this removal.

Reply:
I restored this and updated the internal logic to fix some issues that already existed in canary.